src/services/usersServer.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+
+let users = [{
+  id: uuidv4(), name: 'Vasy',
+}, {
+  id: uuidv4(), name: 'Oleg',
+} ];
```

Comment:
remove it to pass the tests

Reply:
also, try to figure out why you get so many users when there shouldn't be any of them
![image](https://user-images.githubusercontent.com/98764641/236409248-efa19576-e26b-434d-833c-25833b9b9165.png)
maybe you need some initialization

Reply:
same for expenses